script.js

```diff
@@ -0,0 +1,114 @@
+const colors = [
+    "#0fbcf9",
+    "#00d8d6",
+    "#05c46b",
+    "#ffc048",
+    "#ffdd59"
+  
+    ];
+
+  
+    let firstColor = colors[Math.floor(Math.random() * colors.length)];
+    let secondColor = colors[Math.floor(Math.random() * colors.length)];
+    
+    document.body.style.background = `linear-gradient(${firstColor}, ${secondColor})`;
+   
+    const toDoButton = document.getElementById("toDoButton");
+    const toDo = document.getElementById("toDo");
+    const toDoList = document.getElementById("todo-list");
+    const toDoInfo = document.getElementById("toDoInfo");
+    const doneList = document.getElementById("done-list");
+    const doneInfo = document.getElementById("doneInfo");
+    const finishtoDo = document.getElementById("finishtoDo");
```

Comment:
변수들을 한 것에 모아 가독성을 높인 부분 정말 좋다고 생각합니다.
다만 `getElementById`를 해서 가져올 때 [네이밍컨벤션](https://velog.io/@cada/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EC%8A%A4%ED%83%80%EC%9D%BC-%EA%B0%80%EC%9D%B4%EB%93%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-%EC%BB%A8%EB%B2%A4%EC%85%98-%ED%8E%B8)
을 참고해서 작성해 주시면 좋을 것 같습니다!

Reply:
더 가독성 있게 바꿔보겠습니다 ! 좋은 자료 감사합니다

Reply:
현재 님이 걸어 주신 링크에는 나와 있지 않은 것 같은데, HTML 태그의 id/class 이름은 보통 `kebab-case`로 짓습니다!
```suggestion
    const toDoButton = document.getElementById("todo-button");
    const toDo = document.getElementById("todo");
    const toDoList = document.getElementById("todo-list");
    const toDoInfo = document.getElementById("todo-info");
    const doneList = document.getElementById("done-list");
    const doneInfo = document.getElementById("done-info");
    const finishtoDo = document.getElementById("finish-todo");
```